### PR TITLE
Volumes and sa on dc describer

### PR DESCRIPTION
**Pull request by kargakis**
Fixes https://github.com/openshift/origin/issues/8134

New dc description:
```
Name:		docker-registry
Created:	5 hours ago
Labels:		docker-registry=default
Annotations:	<none>
Latest Version:	1
Selector:	docker-registry=default
Replicas:	1
Triggers:	Config
Strategy:	Rolling
Template:
  Labels:	docker-registry=default
  Containers:
  registry:
    Image:	openshift/origin-docker-registry:latest
    Port:	5000/TCP
    QoS Tier:
      cpu:	BestEffort
      memory:	BestEffort
    Liveness:	http-get http://:5000/healthz delay=10s timeout=5s period=10s #success=1 #failure=3
    Readiness:	http-get http://:5000/healthz delay=0s timeout=5s period=10s #success=1 #failure=3
    Environment Variables:
      [ ... ]
      OPENSHIFT_INSECURE:	false
      OPENSHIFT_MASTER:		https://10.0.2.15:8443
      REGISTRY_HTTP_ADDR:	:5000
      REGISTRY_HTTP_NET:	tcp
      REGISTRY_HTTP_SECRET:	XSUsTRwjbvRznNlHqh85d1UVOr6AnaC2GkZxVXREx5o=
  Volumes:
  registry-storage:
    Type:	EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:	
Deployment #1 (latest):
	Name:		docker-registry-1
	Created:	5 hours ago
	Status:		Complete
	Replicas:	1 current / 1 desired
	Selector:	deployment=docker-registry-1,deploymentconfig=docker-registry,docker-registry=default
	Labels:		docker-registry=default,openshift.io/deployment-config.name=docker-registry
	Pods Status:	1 Running / 0 Waiting / 0 Succeeded / 0 Failed
No events.
```

### CODE REVIEW

**Comment by kargakis** [comment]
sa example:
```
Name:		router
Created:	5 hours ago
Labels:		router=router
Annotations:	<none>
Latest Version:	1
Selector:	router=router
Replicas:	1
Triggers:	Config
Strategy:	Rolling
Template:
  Labels:		router=router
  Service account:	router
  Containers:
  router:
    Image:	openshift/origin-haproxy-router:v1.1.4
    Ports:	80/TCP, 443/TCP, 1936/TCP
    QoS Tier:
      memory:	BestEffort
      cpu:	BestEffort
    Liveness:	http-get http://localhost:1936/healthz delay=10s timeout=1s period=10s #success=1 #failure=3
    Readiness:	http-get http://localhost:1936/healthz delay=10s timeout=1s period=10s #success=1 #failure=3
    Environment Variables:
      ROUTER_EXTERNAL_HOST_HOSTNAME:		
      ROUTER_EXTERNAL_HOST_HTTPS_VSERVER:	
      ROUTER_EXTERNAL_HOST_HTTP_VSERVER:	
      ROUTER_EXTERNAL_HOST_INSECURE:		false
      ROUTER_EXTERNAL_HOST_PARTITION_PATH:	
      ROUTER_EXTERNAL_HOST_PASSWORD:		
      ROUTER_EXTERNAL_HOST_PRIVKEY:		/etc/secret-volume/router.pem
      ROUTER_EXTERNAL_HOST_USERNAME:		
      ROUTER_SERVICE_HTTPS_PORT:		443
      ROUTER_SERVICE_HTTP_PORT:			80
      ROUTER_SERVICE_NAME:			router
      ROUTER_SERVICE_NAMESPACE:			default
      ROUTER_SUBDOMAIN:				
      STATS_PASSWORD:				EKgAASdMOO
      STATS_PORT:				1936
      STATS_USERNAME:				admin
  No volumes.
Deployment #1 (latest):
	Name:		router-1
	Created:	5 hours ago
	Status:		Complete
	Replicas:	1 current / 1 desired
	Selector:	deployment=router-1,deploymentconfig=router,router=router
	Labels:		openshift.io/deployment-config.name=router,router=router
	Pods Status:	1 Running / 0 Waiting / 0 Succeeded / 0 Failed
No events.
```

**Comment by smarterclayton** [comment]
Account should be capitalized.  Add a new line between volumes and deployments.  Add a new line between deployments and events

**Comment by kargakis** [comment]
> Account should be capitalized. Add a new line between volumes and deployments. Add a new line between deployments and events

Done.

File: pkg/cmd/cli/describe/deployments.go [hunk 1]
```diff
@@ -248,80 +245,30 @@ func printTriggers(triggers []deployapi.DeploymentTriggerPolicy, w *tabwriter.Wr
...
-			probe := kctl.DescribeProbe(container.ReadinessProbe)
-			fmt.Fprintf(w, "    Readiness:\t%s\n", probe)
-		}
-
-		fmt.Fprintf(w, "    Environment Variables:\n")
-		for _, e := range container.Env {
-			fmt.Fprintf(w, "      %s:\t%s\n", e.Name, e.Value)
-		}
-	}
+	fmt.Fprintln(w)
```
Comment (smarterclayton) [inline pkg/cmd/cli/describe/deployments.go]: I would put this in the code that calls this method - generally you want to control separation visually in the method that brings the sub components together, not in the individual components.

Comment (kargakis) [inline pkg/cmd/cli/describe/deployments.go]: ok

**Comment by smarterclayton** [comment]
Lgtm [merge]

**Pull request merged**